src/components/LogoutLink.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import Link from 'react-router/lib/Link';
+
+/**
+ * Logout link component
+ * @module LogoutLink
+ */
+class LogoutLink extends React.Component {
+	render() {
+		const {
+			to,
+			children,
+			...other
+		} = this.props;
+
+		return (
+			<Link
+				to={{ pathname: to, query: {logout: true} }}
```

Comment:
Hm I don't get the `Invalid cookie value` error, but it does log me out right after a successful login because it's refreshing the `?logout` URL. Send the defect over to me and I can fix it next sprint